quantum/encoder.c

```diff
@@ -56,8 +60,13 @@ __attribute__((weak)) void encoder_update_kb(int8_t index, bool clockwise) { enc
 void encoder_init(void) {
 #if defined(SPLIT_KEYBOARD) && defined(ENCODERS_PAD_A_RIGHT) && defined(ENCODERS_PAD_B_RIGHT)
     if (!isLeftHand) {
+#ifndef ENCODER_DIRECTION_FLIP
         const pin_t encoders_pad_a_right[] = ENCODERS_PAD_A_RIGHT;
         const pin_t encoders_pad_b_right[] = ENCODERS_PAD_B_RIGHT;
+#else
+        const pin_t encoders_pad_a_right[] = ENCODERS_PAD_B_RIGHT;
+        const pin_t encoders_pad_b_right[] = ENCODERS_PAD_A_RIGHT;
+#endif
```

Comment:
Would we ever expect a scenario where Main hand encoders are fine, but only offhand encoders are backwards?

Similarly, should we even have this ENCODER_DIRECTION_FLIP define at all? If the encoders were wired up backwards in the keyboard's config.h shouldn't it just be fixed / flipped there?

Reply:
Honestly, not sure.  But it would generate more consistent behavior, which I think may be more important.  
However, maybe we should have a `_RIGHT` version of the define?